src/windows/lswclient/LSWApi.h

```diff
@@ -205,6 +205,10 @@ HRESULT WslInstallComponents(enum WslInstallComponent Components, WslInstallCall
 // Used for testing until the package is published.
 HRESULT WslSetPackageUrl(LPCWSTR Url);
 
+HRESULT WslMountWindowsFolder(LSWVirtualMachineHandle VirtualMachine, LPCWSTR WindowsPath, const char* LinuxPath, BOOL ReadOnly);
+
+HRESULT WslUnmountWindowsFolder(LSWVirtualMachineHandle VirtualMachine, const char* LinuxPath);
```

Comment:
It's a little odd to take the Linux path for an api to mount a Windows folder.

Reply:
I'm not following, this the API that unmounts the folders. Given that the same Windows paths can be mounted in multiple places in the guest, the linux path is the only way to identify the mount point

Reply:
I mean it's odd for a method called "UnmountWindowsFolder" to take "LinuxPath" as an argument, but I see your point.